### PR TITLE
Adds eslint

### PR DESCRIPTION
**Pull request by arnoldsandoval-okta**
- Adds eslint to monorepo
- Fixes linter errors

### CODE REVIEW

File: packages/vuepress-theme-odyssey/templates/DocsTemplateHome.vue [hunk 1]
```diff
@@ -4,12 +4,24 @@
...
-        <p  class="docs-hero--desc">{{ $page.frontmatter.lead }}</p>
+        <p class="docs-hero--desc">{{ $page.frontmatter.lead }}</p>
       </div>
-      <div class="docs-hero-moon" v-html="require(`!html-loader!../../docs/.vuepress/public/images/home-hero-moon.svg`)"></div>
-      <div class="docs-hero-rocket" v-html="require(`!html-loader!../../docs/.vuepress/public/images/home-hero-rocket.svg`)"></div>
+      <!-- eslint-disable vue/no-v-html -->
+      <div
+        class="docs-hero-moon"
+        v-html="
+          require(`!html-loader!../../docs/.vuepress/public/images/home-hero-moon.svg`)
```
Comment (jmelberg-okta) [inline packages/vuepress-theme-odyssey/templates/DocsTemplateHome.vue:13]: Are there alternatives to using the html-loader? Would be nice to not have to add `eslint-disable` in all these files.

File: packages/vuepress-theme-odyssey/global-components/Visual.vue [hunk 2]
```diff
@@ -27,35 +24,39 @@ export default {
...
+  data: function() {
     return {
       labels: {
-        'positive': 'Do',
-        'negative': `Don't`,
-        'smiley-positive': 'Best!',
-        'smiley-neutral': 'Good',
-        'smiley-negative': 'Bad'
+        positive: "Do",
+        negative: `Don't`,
```
Comment (jmelberg-okta) [inline packages/vuepress-theme-odyssey/global-components/Visual.vue:54]: nit: This can be double quotes

```suggestion
        negative: "Don't",
```

**Review by jmelberg-okta — APPROVED** [review]

**Review by edburyenegren-okta — APPROVED** [review]
just did a run through the site; looks good, no regressions from the fixes! all of our JS examples are still working as well.

you may want to rebase against `develop` so the tests run against the new Table content and component changes as well.

File: packages/vuepress-theme-odyssey/templates/DocsTemplateHome.vue (hunk 1 above)
Comment (arnoldsandoval-okta) [reply packages/vuepress-theme-odyssey/templates/DocsTemplateHome.vue:13]: I 100% agree, I will log an issue to investigate and replace these.

Comment (arnoldsandoval-okta) [reply packages/vuepress-theme-odyssey/templates/DocsTemplateHome.vue:13]: Logged #809

**Pull request merged**